src/StudyViewer/SingleStudyViewer.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { Space, Typography, Spin } from 'antd';
+import { FileOutlined, FilePdfOutlined } from '@ant-design/icons';
+import BackLink from '../components/BackLink';
+import { humanFileSize } from '../utils.js';
+import { ReduxStudyDetails, fetchDataset, fetchFiles } from './reduxer';
+import getReduxStore from '../reduxStore';
+import './StudyViewer.css';
+
+const { Title } = Typography;
+
+class SingleStudyViewer extends React.Component {
+  componentDidMount() {
+    getReduxStore().then(
+      store =>
+        Promise.all(
+          [
+            store.dispatch(fetchDataset(decodeURIComponent(this.props.match.params[0]))),
+            store.dispatch(fetchFiles('object', decodeURIComponent(this.props.match.params[0]))),
+            store.dispatch(fetchFiles('open-access', decodeURIComponent(this.props.match.params[0]))),
+          ],
+        ));
+  }
+
+  render() {
+    if (!this.props.dataset || this.props.dataset.length === 0) {
+      return (
+        <div className='study-viewer'>
+          <div className='study-viewer_loading'>
+            <Spin size='large' tip='Loading data...' />
+          </div>
+        </div>
+      );
+    }
+    const dataset = this.props.dataset[0];
+    return (
+      <div className='study-viewer'>
+        <BackLink url='/study-viewer' label='Back' />
+        <Space className='study-viewer__space' direction='vertical'>
+          <div className='study-viewer__title'>
+            <Title level={4}>{dataset.title}</Title>
+          </div>
+          <div className='study-viewer__details'>
+            <ReduxStudyDetails data={dataset} fileData={this.props.fileData} />
+            <div className='study-viewer__details-sidebar'>
+              <Space direction='vertical' style={{ width: '100%' }}>
+                <div className='study-viewer__details-sidebar-box'>
+                  <Space className='study-viewer__details-sidebar-space' direction='vertical'>
+                    <div className='h3-typo'>Data Access Agreements</div>
+                    <div>
+                      <FilePdfOutlined />
+                      <a href=''>Data Use Agreement (DUA)</a>
+                    </div>
+                    <div>
+                      <FilePdfOutlined />
+                      <a href=''>Data Access Request (DAR)</a>
+                    </div>
```

Comment:
idk if we'd want to merge this to prod 😬

Reply:
probably not, will remove